### ISSUE TITLE
Dashboard survey

### ISSUE THREAD

**Issue by danielromlein** [kubernetes/dashboard#3256]
tl;dr @spencersugarman and I would like to conduct a survey around Dashboard and K8s UIs generally, and we want your input on what questions would be valuable to ask. 

### Please check out our [draft survey](https://docs.google.com/document/d/1C18n2Bhyc1gxeKRJfj9qhNkIGSoWUZchBXFp4c43VYQ/edit?usp=sharing) and comment with any suggestions on how it could be improved.

The last significant Dashboard survey we ran was in November 2016, and the responses garnered (over 100) was used to inform roadmap decisions more in-line with Dashboard's users. The results we summarized in a [blog post](https://kubernetes.io/blog/2017/01/kubernetes-ux-survey-infographic/).

Our plan is to refine the survey, send it out and promote it on Slack / Twitter / kubernetes-sig-ui@googlegroups.com mailing list in the coming weeks, then have preliminary findings & subsequent recommendations to share as part of our [KubeCon talk](http://sched.co/FuKi) this November. At [KubeCon China](https://events.linuxfoundation.cn/events/kubecon-cloudnativecon-china-2018/) and [KubeCon North America](https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/) we'll further promote and gather more data.

Thanks!

**Comment by spencersugarman** [comment]
Last call for comments! We will be finalizing the survey by the end of the week, and sharing out early next week.

**Comment by spencersugarman** [comment]
Survey is published (in English and Chinese)! If you have 10 minutes, please it: https://google.qualtrics.com/jfe/form/SV_0xGSwxVpJNnT89f

Thanks to everyone who provided feedback, and special thanks to @pengx17 for helping translate the survey. If you have a community you want to share with, feel free to send the survey link around, or retweet Dan: https://twitter.com/danielromlein/status/1044779215504257030

**Comment by maciaszczykm** [comment]
Seems like the survey is not active anymore. Closing it to keep issue number lower since the discussion is done. Feel free to reopen.

/close

**Comment by spencersugarman** [comment]
Yes, survey is closed now. Thank you to everyone who replied. To show our appreciation, we will be sending out codes to 5 lucky random respondents for a free Kubernetes hoodie! 

The raw data is available in Microsoft Excel format ([KubeCon China Dashboard Survey - Responses.xlsx](https://github.com/kubernetes/dashboard/files/2564048/KubeCon.China.Dashboard.Survey.-.Responses.xlsx)) or on [Google Sheets](https://docs.google.com/spreadsheets/d/1LdxQTG4Rn5PagZed2yR9G8RNkUVB4xdc7Im8XPwAfNM/edit?usp=sharing) (feel free to comment!).

@danielromlein and I will be presenting the main results at KubeCon China next Thursday (November 15) at 16:45: [State of the UI: Leveraging Kubernetes Dashboard and Shaping its Future](https://kccncchina2018english.sched.com/event/FuKi/state-of-the-ui-leveraging-kubernetes-dashboard-and-shaping-its-future-dan-romlein-spencer-sugarman-google). We will also be hanging out at the Google booth in the sponsor hall. If you are joining, come say Hi!

**Comment by danielromlein** [comment]
@spencersugarman could you attach here the PDF of the survey results with charts?
/ @maciaszczykm @floreks in case the two of you have a chance to review before our [SIG-UI working session 12.10](https://calendar.google.com/event?action=TEMPLATE&tmeid=Mmw1cmt1aG83OW0yYW5kaHVudmI0ZzUycXYgZHJvbWxlaW5AZ29vZ2xlLmNvbQ&tmsrc=dromlein%40google.com)

**Comment by spencersugarman** [comment]
Here is the full report:
[KubeCon China Dashboard Survey - Report.pdf](https://github.com/kubernetes/dashboard/files/2661909/KubeCon.China.Dashboard.Survey.-.Report.pdf). 

I can also dig into the raw data (posted in my previous comment) if anyone is interested in seeing the results sliced in different ways.
